CHANGELOG.md

```diff
@@ -1,16 +1,28 @@
-# Change Log for AWS IoT Device SDK for Embedded C
+# Changelog for AWS IoT Device SDK for Embedded C
+
+## 202011.00 (November 2020)
+
+### Major Changes
+
+- This release introduces the re-factored **HTTP** client library, **AWS IoT Device Defender** service client library, and **AWS IoT Jobs** service library.
+These libraries have gone through code quality checks including verification that no function has a [GNU Complexity](https://www.gnu.org/software/complexity/manual/complexity.html) score greater than 8, and checks against deviations from the mandatory rules in the [MISRA coding standard](https://www.misra.org.uk/MISRAHome/MISRAC2012/tabid/196/Default.aspx). These libraries have also undergone both static code analysis from [Coverity statical analysis](https://scan.coverity.com/) and validation of memory safety through the [CBMC automated reasoning tool](https://www.youtube.com/watch?v=YwQHAPRhQkI&feature=youtu.be&t=1721).
```

Comment:
Do we need to post the CBMC video? Or should we use the link: http://www.cs.cmu.edu/~modelcheck/cbmc/

Reply:
We can use that link instead